nodes/src/nodes/telegram/IEndpoint.py

```
@@ -0,0 +1,410 @@
+                    timeout=aiohttp.ClientTimeout(total=35),
+                ) as resp:
+                    data = await resp.json()
+
+                poll_count += 1
+                ok = data.get('ok')
+                updates = data.get('result', []) if ok else []
+
+                if ok:
+                    for update in updates:
+                        uid = update.get('update_id')
+                        asyncio.create_task(self._handle_update(update))
```

Ruff (0.15.9)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)